lib/twitter-ads/http/ton_upload.rb

```diff
@@ -10,13 +10,13 @@ class TONUpload
     DEFAULT_RESOURCE = '/1.1/ton/bucket/'.freeze # @api private
     DEFAULT_BUCKET   = 'ta_partner'.freeze # @api private
     DEFAULT_EXPIRE   = (Time.now + 10 * 24 * 60 * 60).httpdate # @api private
-    MIN_FILE_SIZE    = 1024 * 1024 * 64 # @api private
+    MAX_FILE_SIZE    = 1024 * 1024 * 64 # @api private
```

Comment:
This line's value has been changed by https://github.com/twitterdev/twitter-ruby-ads-sdk/commit/d55b2c25c99236c32acf7ad5b0a7ad57b06ae920 from 1MB to 64MB.
The [official document](https://developer.twitter.com/en/docs/ads/audiences/overview/ton-upload) said

> Some uploads such as videos or large tailored audiences might be greater than 64MB and should use resumable upload.

so, I think it's `MAX_FILE_SIZE` not `MIN_FILE_SIZE` .